test/extended/testdata/builds/pullsecret/pullsecret-nodejs-bc.yaml

```diff
@@ -11,6 +11,6 @@ spec:
     sourceStrategy:
       from:
         kind: DockerImage
-        name: registry.redhat.io/rhscl/nodejs-10-rhel7:latest
+        name: image-registry.openshift-image-registry.svc:5000/openshift/nodejs:10
```

Comment:
@smarterclayton .... per our discussion in slack yesterday ... see https://coreos.slack.com/archives/C014MHHKUSF/p1605016754450000?thread_ts=1604938369.426700&cid=C014MHHKUSF

and https://github.com/openshift/origin/pull/25675#issuecomment-724730404

where you going to add an exclusion for `registry.redhat.io/rhscl/nodejs-10-rhel7:latest` ... and as a result, revert this particular change?

otherwise, the associated test will still fail, as it did today with https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/24887/pull-ci-openshift-origin-master-e2e-gcp-builds/1326569182518579200

Reply:
Yes, was wrestling with other failures yesterday (working around the issue with KUBE_TEST_REPO_LIST being added).